### ISSUE TITLE
Update output netcdf units attribute to include correct time denomination

### ISSUE THREAD

**Issue by tobeycarman** [ua-snap/dvm-dos-tem#336]
Currently the units attribute all variables looks like this:

```
$ ncdump -h run-1/out/ENS-00001/GPP_yearly_tr.nc 
netcdf GPP_yearly_tr {
dimensions:
	time = UNLIMITED ; // (109 currently)
	y = 10 ;
	x = 10 ;
variables:
	double GPP(time, y, x) ;
		GPP:units = "gC/m2/time" ;
...
```

For working with PEcAn (and to better conform to CF standards), it would be really helpful if we could have the units be per year, per month, and per day instead of per time, e.g. `"gC/m2/year"`

**Comment by rarutter** [comment]
Currently the units string is pulled straight from the output_spec.csv file, so this could be "fixed" by the user replacing 'time' with their selected timestep. When Issue #331 is addressed, this will have to be automatic for each timestep.

**Comment by rarutter** [comment]
Addressed by PR #350
